website/docs/cloud-docs/policy-enforcement/define-policies/opa.mdx

```diff
@@ -4,34 +4,36 @@ description: Use the Rego policy language to define Open Policy Agent (OPA) poli
 
 ---
 
-# Defining OPA Policies
+# Define OPA policies 
 
-Policies are rules that HCP Terraform enforces on runs. You use the [Rego policy language](https://www.openpolicyagent.org/docs/latest/policy-language/) to write policies for the Open Policy Agent (OPA) framework. After you define policies, you must add them to [policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) that HCP Terraform can enforce globally or on specific [projects](/terraform/cloud-docs/projects/manage) and workspaces.
-OPA Policies are evaluated in HCP Terraform's infrastructure.
+This topic describes how to create and manage custom policies using the open policy agent (OPA) framework. Refer to the following topics for instructions on using HashiCorp Sentinel policies:
+
+[Define custom Sentinel policies](/terraform/cloud-docs/policy-enforcement/define-policies/custom-sentinel)
+[Copy pre-written Sentinel policies](/terraform/cloud-docs/policy-enforcement/define-policies/prewritten-sentinel)
 
 <!-- BEGIN: TFC:only name:pnp-callout -->
```

Comment:
Are these still true now that RUM pricing is live?

Reply:
Yeah, looks like it according to the pricing page.